src/renderer/wg_engine/tvgWgGeometry.h

```diff
@@ -208,15 +207,24 @@ struct WgVertexBuffer
                 pntIndex++;
             } else if (cmd == PathCommand::Close) {
                 close();
+                // close command creates the new object, started form the closing point
+                if (update_dist) updateDistances();
+                if ((vcount > 0) && (onPolyline)) onPolyline(*this);
+                Point last_pt = last();
+                reset(tscale);
+                // append new point in a case if close command is not the last command and next command is LineTo
+                if ((cmdIndex < rshape.path.cmds.count - 1) && (rshape.path.cmds[cmdIndex + 1] == PathCommand::LineTo))
+                    append(last_pt);
```

Comment:
@SergeyLebedkin from the beggining I have the same comment: why it can not be done as below?
```
if ((cmdIndex < rshape.path.cmds.count - 1) && (rshape.path.cmds[cmdIndex + 1] != PathCommand::MoveTo)) {
    if (update_dist) updateDistances();
    if ((vcount > 0) && (onPolyline)) onPolyline(*this);
    reset(tscale);
    append(last());
}
```
I understand that you want to handle cases different than 'Close MoveTo' (so either 'Close LineTo' or 'Close CubicTo'). For a standard case 'Close MoveTo' nothing has to be changed, right? it worked till now correct, so there is no need to call neither the updateDistances() nor the onPolyline()? Or these were missing and this pr solves 2 problems?

Reply:
We need to handle cases, when after Close Command we have a command like LineTo or CubicTo.
We don`t handle this cases till now.
In this cases we need to proceed path (updateDist() and OnPolylone()) and restart new object path from the closed point

Reply:
I understand this, that's why I'm asking why don;t you add a check 'if ((cmdIndex < rshape.path.cmds.count - 1) && (rshape.path.cmds[cmdIndex + 1] != PathCommand::MoveTo))' just after close() is called

Reply:
@mgrudzinska so qustion is closed?

Reply:
@mgrudzinska and you was right with a CubicTo case. So I also add this case